src/main/java/com/example/codeE/util/DateTimeUtil.java

```diff
@@ -11,4 +11,7 @@ public class DateTimeUtil {
     public static String format(LocalDateTime dateTime) {
         return dateTime.format(formatter);
     }
+    public static String getDateNowbyformat(String format){
```

Comment:
What does getDateNowbyformat mean? Please name the function in a grammatically correct and understandable way :((((